bin/physlr-make

```diff
@@ -103,13 +103,13 @@ arcs=false
 .PHONY: f1chr4 f1chr2R f1 fishchr25 fish physical-map scaffolds
 all: f1chr4 f1chr2R f1 fishchr25 fish
 
-$(lr).physlr.physical-map.path: $(lr).k$k-w$w.n$(minimum_barcode_multiplicity)-$(maximum_barcode_multiplicity).c2-x.physlr.overlap.m$m.mol.mol2-bcs.backbone.path
+$(lr).physlr.physical-map.path: $(lr).k$k-w$w.n$(minimum_barcode_multiplicity)-$(maximum_barcode_multiplicity).c2-x.physlr.overlap.m$m.ext.mol.backbone.path
```

Comment:
What is the `ext` stand for, and it it necessary?

Reply:
Is it necessary? No, but it tells us what the type of mol separation that is used which is distributed+sqcosbin

Reply:
So what are the other options other than `ext` that are possible?

Reply:
"					bc (biconnected componenets) is the least conservative and is only suitable for datasets with low barcode multiplicity."
 "					bc+k3 (biconnected componenets + k-3 cliques) is more conservative than bc and requires more time."
 "					distributed is a modified version of bc+k3 that is faster than bc+k3 but may be more (or even less) conservative."
 "					ext (extensive) mixes distributed with a modified version of sqcos (cosine similarity of squared adjacency matrix) which makes it more conservative."

Reply:
`bc` ->`bc.mol` (is not in the makefile anymore)
`k3`->`k3.mol`,
`distributed`->`mol`,
`distributed+sqcosbin`->`ext`
`distributed+sqcosbin++sqcosbin++sqcosbin` -> `iterative.mol` (will be added with my recent PR)

`ext` or `iterative` may become the default molecule separation strategy we use, as they produce more reliable outputs.

Reply:
Thanks for the clarification. 
Shouldn't this then use the `mol_strategy` variable?

Reply:
`ext` is always `distributed+sqcosbin`. So no need to use `mol_strategy`.

In case hte user want to test his/her preferred strategy to make a phycal map or assembly he can use `mol_strategy` and there is a rule for that.
More exactly, the rule is not `distributed` -> `mol`, it is `mol_strategy` -> `mol` and the default value for `mol_strategy` is `distributed`. So the user can use this rule.

Reply:
But how it was currently for these lines, the `.path` output would always soft-link to the file with `ext`, wouldn't it?
Looks like you made a fix, Johnathan?

Reply:
The problem is the physical map rules use ext. Its been modified to use mol and the default value is now `distributed+sqcosbin`.

Reply:
... unless we want to make it the default for `.mol.`. So I changed my mind and yes we can use the variable.

Reply:
Awesome -- the change looks good to me!